api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -286,7 +285,7 @@ private void updateInstitutionalAffiliations(Profile updatedProfile, DbUser user
     }
   }
 
-  private void validateUpdatedProfile(Profile updatedProfile, Profile prevProfile)
+  public void validateUpdatedProfile(Profile updatedProfile, Profile prevProfile)
```

Comment:
I feel like having to expose something like this is a code smell. This should be a guarantee of the profile service: it always returns valid profiles or throws. I shouldn't have to remember to call a validator (or which validator to call).

Granted, you inherited this issue and the night is young.